<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1504</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Display_Window">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>571</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Display_Window_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Display_Window_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:yellow;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hue Thresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sat Thresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BeginPaintButt">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>520</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Paint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveDataButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>560</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="canvasLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>117</y>
      <width>91</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>117</width>
      <height>500</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="blackbutton">
       <property name="text">
        <string>Black</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="whitebutton">
       <property name="text">
        <string>White</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redbutton">
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bluebutton">
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="greenbutton">
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yellowbutton">
       <property name="text">
        <string>Yellow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="invertButton">
       <property name="text">
        <string>Invert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blackWhitebutton">
       <property name="text">
        <string>Black/White</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="blackSelectedLabel">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>8</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="invertWhiteOutline">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>367</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueSelectedLabel">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>185</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueLabel">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>192</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: blue;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="yellowWhiteOutline">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>308</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="redLabel">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>133</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blackWhiteOutline">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>427</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="redWhiteOutline">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>131</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="redSelectedLabel">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>126</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="yellowSelectedLabel">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>303</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="redOutline">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>132</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blackWhiteWhiteOutline">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>426</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueOutline">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>191</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="invertSelectedLabel">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>362</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="whiteWhiteOutline">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>72</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blackLabel">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>15</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="whiteLabel">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>74</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: white;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="whiteSelectedLabel">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>67</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blackWhiteSelectedLabel">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>421</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blackOutline">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>14</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="greenLabel">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>251</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: green;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="greenWhiteOutline">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>249</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="yellowLabel">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: yellow;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blackWhiteBorder">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>13</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="invertLabel">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>369</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/invert.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="greenOutline">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>250</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="whiteOutline">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>73</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="invertOutline">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>368</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="greenSelectedLabel">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>244</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blackWhiteLabel">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>428</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/blackwhite.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="yellowOutline">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>309</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueWhiteOutline">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>190</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BackgroundMask">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>610</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:purple;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Display_Window</zorder>
   <zorder>Display_Window_2</zorder>
   <zorder>Display_Window_5</zorder>
   <zorder>verticalSlider</zorder>
   <zorder>verticalSlider_2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>BeginPaintButt</zorder>
   <zorder>saveDataButton</zorder>
   <zorder>canvasLabel</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>blackSelectedLabel</zorder>
   <zorder>blueSelectedLabel</zorder>
   <zorder>redSelectedLabel</zorder>
   <zorder>yellowSelectedLabel</zorder>
   <zorder>invertSelectedLabel</zorder>
   <zorder>whiteSelectedLabel</zorder>
   <zorder>blackOutline</zorder>
   <zorder>blackWhiteBorder</zorder>
   <zorder>greenSelectedLabel</zorder>
   <zorder>blackWhiteLabel</zorder>
   <zorder>blueWhiteOutline</zorder>
   <zorder>blackLabel</zorder>
   <zorder>whiteWhiteOutline</zorder>
   <zorder>whiteOutline</zorder>
   <zorder>whiteLabel</zorder>
   <zorder>redWhiteOutline</zorder>
   <zorder>redOutline</zorder>
   <zorder>redLabel</zorder>
   <zorder>blueOutline</zorder>
   <zorder>blueLabel</zorder>
   <zorder>greenWhiteOutline</zorder>
   <zorder>greenOutline</zorder>
   <zorder>greenLabel</zorder>
   <zorder>yellowWhiteOutline</zorder>
   <zorder>yellowOutline</zorder>
   <zorder>yellowLabel</zorder>
   <zorder>invertWhiteOutline</zorder>
   <zorder>invertOutline</zorder>
   <zorder>invertLabel</zorder>
   <zorder>blackWhiteSelectedLabel</zorder>
   <zorder>blackWhiteWhiteOutline</zorder>
   <zorder>blackWhiteOutline</zorder>
   <zorder>BackgroundMask</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1504</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
